### ISSUE TITLE
remote control  screenshare UI issues

### ISSUE THREAD

**Issue by Tuxicoman2** [jitsi/jitsi-meet-electron#153]
Hi,

I have 2 UI issues with the current status when controling a remote desktop:

1. the "JITSI.org" logo in overlay on the upper left side of the "jitsi-meet" window is clickable and opens a webbrowser. Therefore, it's impossible to use this area on the remote desktop. Unfortunately this is the area where the main menus of any app like will show :-(
1. When the mouse goes to the bottom of the"jitsi-meet" window, the bottom bar with the jisti controls show up in overlay on top of the remote desktop. This means the bottom area of the remote desktop is not available for control because I can't click on it (the "jitsi-meet" control bar always comes in between). Unfortunately, this the area where OSX dock is.

So maybe a solution is ton provide a way to keep the remote desktop area clean of any overlay for jitsi-meet ui.
Also be carefull to not implement magnetic sides (bringing mouse close to a side would popup a UI element) in this mode. As it would interfere with the magnetic sides within the remote desktop. IE: OSX dock bar with auto-hide enabled so that it appears after a 2sec delay when the mouse is just near the bottom side of the screen.

**Comment by tuxayo** [comment]
Same issue.

Mitigation: resize the window of the "controller" so that it has gray strips at the top and bottom. The Jitsi Meet controls and logo will be there and not in the video stream anymore.

Put your window in a square shape and you will see what I mean?

**Comment by kael-shipman** [comment]
I'm very interested in seeing a fix for this as well. @tuxayo's workaround is much appreciated and works for now, but having immersive full-screen would be really helpful for all the times I've gotta hop on to help my mom out with stuff ;). I'm trying to start a cash bounty for this on BountySource, but their site is ground to a halt right now. I'll try again later.

**Comment by kael-shipman** [comment]
Ok, I've just posted this to bounty source and am adding a $50 bounty on it (I encourage others to put some money on it as well). However, I'd like to further specify the proposed solution. Obviously we need the UI controls at some point, so I propose the following:

1. On entering remote control mode, master's UI controls are hidden with a pop-up that says "to toggle UI controls, type ctrl+windows (or something similar, perhaps configurable in jitsi electron settings).
2. When user hits key combo, controls show up and key combo is NOT sent to slave
3. When user hits key combo again, controls go away and key combo is NOT sent to slave
4. When user leaves remote control mode, UI controls re-appear.

**Comment by betoreyez** [comment]
> Ok, I've just posted this to bounty source ([here](https://www.bountysource.com/issues/67739777-remote-control-screenshare-ui-issues)) and am adding a $50 bounty on it (I encourage others to put some money on it as well). However, I'd like to further specify the proposed solution. Obviously we need the UI controls at some point, so I propose the following:
> 
> 1. On entering remote control mode, master's UI controls are hidden with a pop-up that says "to toggle UI controls, type ctrl+windows (or something similar, perhaps configurable in jitsi electron settings).
> 2. When user hits key combo, controls show up and key combo is NOT sent to slave
> 3. When user hits key combo again, controls go away and key combo is NOT sent to slave
> 4. When user leaves remote control mode, UI controls re-appear.

The link to the bounty is not working, I like the idea too.

**Comment by kael-shipman** [comment]
Yes, unfortunately, bountysource has huge issues with scaling. Their website is frequently overloaded and down. I've just checked the link and it looks like it's good now. Feel free to try again.

**Comment by betoreyez** [comment]
I've added another 50 bucks in bounty source.
